src/AspNetCoreModuleV2/CommonLib/PollingAppOfflineApplication.cpp

```diff
@@ -45,89 +36,16 @@ PollingAppOfflineApplication::AppOfflineExists()
             m_fAppOfflineFound = is_regular_file(m_appOfflineLocation);
```

Comment:
In the code above, the comment says `1 second` but `c_appOfflineRefreshIntervalMS` is `200ms`.